api.bs

```diff
@@ -855,6 +864,52 @@ if the user has opted out of collection of diagnostic data.
 * User ability to view the impression store and past report submissions.
 
 
+# HTTP API # {#http-api}
+
+\`<dfn http-header><code>Save-Impression</code></dfn>\` is a
+[=structured header/dictionary|Dictionary Structured Header=]
+set on a response requesting that the user agent invoke the
+<a method for=PrivateAttribution>saveImpression()</a> API.
```

Comment:
Somewhat related to https://github.com/w3c/ppa/pull/106#discussion_r1971560202, do we need to check that the site in Step 2.1 of https://w3c.github.io/ppa/#save-impression-api-operation is using `https://` as its scheme? The JS methods are already marked `[SecureContext]`, but if we need to enforce the use of `HTTPS` then we should probably check that in the core `save impression` and `measure conversion` algorithms for consistency.